SPECS/perl-Sys-Virt/perl-Sys-Virt.spec

```diff
@@ -161,4 +161,4 @@ perl Build.PL installdirs=vendor
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_28_Mass_Rebuild
 
 * Tue Jan 23 2018 Daniel P. Berrange <berrange@redhat.com> - 4.0.0-1
-- Update to 4.0.0 release
+- Update to 4.0.0 release
```

Comment:
Is this change intentional? looks like the EOF newline is removed, which I don't think we want

Reply:
see https://github.com/microsoft/CBL-Mariner/pull/7147